app.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "cds-hooks-sandbox",
+  "success_url": "/",
+  "scripts": {},
+  "env": {
+    "NODE_ENV": {
+      "description": "development environment",
+      "value": "development"
+    },
+    "NPM_CONFIG_PRODUCTION": {
+      "description": "development environment",
```

Comment:
Should the description be "production environment" ?

Reply:
@shriniketsarkar Yup, addressed